client/src/components/DataFiles/DataFilesProjectFileListing/DataFilesProjectListing.scss

```diff
@@ -0,0 +1,4 @@
+.shared-workspace__description {
```

Comment:
will it being used in shared-workspace occur in this branch or in a follow on PR